test/stream.test.js

```diff
@@ -127,3 +129,28 @@ test('onSend hook stream', t => {
     fastify.close()
   })
 })
+
+test('disconnecting from a stream response should not crash the server', t => {
+  t.plan(3)
+  const fastify = Fastify()
+
+  fastify.get('/', (req, reply) => {
+    const endlessStream = new stream.Readable({
+      read (size) {
+        this.push(crypto.randomBytes(size))
```

Comment:
this is likely to trigger something close to an endless loop. Can you just push a value once, and not for every call to `read`? This might fix AppVeyor.